packager/media/formats/webm/segmenter.h

```diff
@@ -139,7 +139,8 @@ class Segmenter {
   uint64_t progress_target_ = 0;
   uint64_t accumulated_progress_ = 0;
   uint64_t first_timestamp_ = 0;
-  int64_t sample_duration_ = 0;
+  int64_t sample_durations_[2] = {0, 0};
+  int64_t num_samples_ = 0;
```

Comment:
s/int64_t/size_t/

Reply:
Should num_samples be size_t? I'm assuming only sample_durations_[2] needs to be size_t

Reply:
num_samples_ should be size type, so it should be `size_t`. sample_durations_ should still be int64_t.